src/ir/proto_to_ir.h

```diff
@@ -0,0 +1,122 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+#ifndef SRC_IR_PROTO_TO_IR_H_
+#define SRC_IR_PROTO_TO_IR_H_
+
+#include "src/ir/attributes/attribute.h"
+#include "src/ir/block_builder.h"
+#include "src/ir/ir_context.h"
+#include "src/ir/module.h"
+#include "src/ir/proto/raksha_ir.pb.h"
+#include "src/ir/value.h"
+#include "src/ir/ssa_names.h"
+
+namespace raksha::ir {
+
+using ResultType =
```

Comment:
Too common a name to be in `raksha::ir` namespace. Move it inside `ProtoToIR` class?

Reply:
Ah, that's unused. Removed